### ISSUE TITLE
UDP or ESPHome

### ISSUE THREAD

**Issue by uKL** [Arnold-n/P1P2MQTT#136]
Hello,
Are you considering adding support for UDP based communication or creating ESP Home based configuration?

**Comment by Arnold-n** [comment]
Yes, but several other topics have higher priority, so sorry but this may take a while.

**Comment by DiabloVmax1200** [comment]
uKL for what do you plan to use UDP based communication? Because of Loxone integration? If yes Loxberry with MQTT Sever works pretty. I also first thought UDP would be great to have. But the MQTT integration has huge benefit for Loxone. https://wiki.loxberry.de/konfiguration/widget_help/widget_mqtt

**Comment by uKL** [comment]
> uKL for what do you plan to use UDP based communication? Because of Loxone integration? If yes Loxberry with MQTT Sever works pretty. I also first thought UDP would be great to have. But the MQTT integration has huge benefit for Loxone. https://wiki.loxberry.de/konfiguration/widget_help/widget_mqtt

Yes, for Loxone. I'd be using MQTT for now but for stability I'd like to make it independent of my Loxberry Instance like I have it for other ESP32 based sensors.

**Comment by DiabloVmax1200** [comment]
Hi. Where do you have stability issues? I’ve the same setup but actually have the feeling that it runs stable

**Comment by uKL** [comment]
> Hi. Where do you have stability issues? I’ve the same setup but actually have the feeling that it runs stable

In general it is stable. Conceptually, I’m having an idea of making critical items like security, heating, electricity, independent of Loxberry - just in case.

**Comment by DiabloVmax1200** [comment]
Hi, this makes sense. Less parts more reliable.
In my configuration I only would lose some comfort and I might would use a bit more energy.

**Comment by uKL** [comment]
> Hi, this makes sense. Less parts more reliable. In my configuration I only would lose some comfort and I might would use a bit more energy.

Well, it depends. If the heatpump would be stuck in DHW heating disabled, this means that there's no hot water in the shower ;-) Wife acceptance factor = -1

... and his had happened with the cloud integration for a few times.

**Comment by DiabloVmax1200** [comment]
That’s definitely an issue ☺️

**Comment by Gorgar91** [comment]
I purchased "older" model just because of lack of UDP functionality, using budulinek setup now with loxone, but if UDP will come into place , I am ready to do purchase again .  Thanks Arnold for all this !
